.github/workflows/pr.yaml

```diff
@@ -0,0 +1,79 @@
+on: pull_request
+name: Review
+jobs:
+  'assets-coding-standards-pratchett':
```

Comment:
Please remove quotes around the job label (for all labels).

Reply:
Fixed.